src/tests/framework/Framework_Stability_003.test.js

```diff
@@ -0,0 +1,152 @@
+import {
+  setWebKitUrl,
+  startHttpServer,
+  matchIpRange,
+  webKitBrowserStartAndResume,
+} from '../../commonMethods/commonFunctions'
+
+let listener
+
+export default {
+  title: 'Framework stability set URL test',
+  description:
+    'Stress loads the system by setting the URL in a loop and see if the Framework process continues to operate nominally',
+  setup() {
+    return this.$sequence([
+      () => webKitBrowserStartAndResume.call(this),
+      () =>
+        (listener = this.$thunder.api.WebKitBrowser.on('urlchange', data => {
+          this.$data.write('currentUrl', data.url)
+        })),
+      () => this.$data.write('app', this.$context.read('html')),
+    ])
+  },
+  context: {
+    html:
+      '<html><head><title>Hello! This is a set url test</title></head><body><p style="position: absolute; left: 150px; top: 150px">Framework Set URL test</p><script>console.log("Page loaded succesfully")</script></body></html>',
+  },
+  steps: [
+    {
+      description: 'Start HTTP server1, serve the app',
+      test() {
+        this.$data.write('appLoaded1', true)
+        startHttpServer.call(this)
```

Comment:
Should we replace this with a call to an html page on the CDN (so this will also work in browser version)?